x-pack/plugins/ml/public/application/components/help_popover/help_popover.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { ReactNode } from 'react';
+import {
+  EuiIcon,
+  EuiLink,
+  EuiLinkButtonProps,
+  EuiPopover,
+  EuiPopoverProps,
+  EuiPopoverTitle,
+  EuiText,
+} from '@elastic/eui';
+import './help_popover.scss';
+
+export const HelpPopoverButton = ({ onClick }: { onClick: EuiLinkButtonProps['onClick'] }) => {
+  return (
+    <EuiText size="xs">
+      <EuiLink onClick={onClick}>
+        <EuiIcon className="mlHelpPopover__buttonIcon" size="s" type="help" />
+      </EuiLink>
+    </EuiText>
```

Comment:
Did you have a reason for using these components instead of an `EuiButtonIcon`? https://elastic.github.io/eui/#/navigation/button#button-icon

Reply:
Good catch, the Lens code I copied it from had additional text which wasn't necessary in this case. I've made that change in c9ddb1804501730bfedbb185b0f58f139c87071c